src/model/ThunderstoreMod.ts

```diff
@@ -16,35 +16,36 @@ export default class ThunderstoreMod extends ThunderstoreVersion implements Reac
     private hasNsfwContent: boolean = false;
     private donationLink: string | undefined;
 
-    public parseFromThunderstoreData(data: any): ThunderstoreMod {
-        this.setName(data.name);
-        this.setFullName(data.full_name);
-        this.setOwner(data.owner);
-        this.setDateCreated(data.date_created);
-        this.setDateUpdated(data.date_updated);
-        this.setDeprecatedStatus(data.is_deprecated);
-        this.setPinnedStatus(data.is_pinned);
+    public static parseFromThunderstoreData(data: any): ThunderstoreMod {
+        const mod = new ThunderstoreMod();
```

Comment:
I'm going to assume this hasn't had any side-effects, was there any reason for converting this to a static method?

Reply:
There's no reasons for the conversions that aren't up to debate. Main reason was that it works nice with [this implementation](https://github.com/ebkr/r2modmanPlus/pull/854/commits/a79a158b0df9599128c49e4e9845392330096973#diff-aaddca80dd3a9609301da14f24a640a5cc268f2de2f6155e5860127ea4d01252L35-R30), which is the only place where the method is used. Also it was my impression that the static approach is favored in the codebase (e.g. [ExportMod](https://github.com/ebkr/r2modmanPlus/blob/develop/src/model/exports/ExportMod.ts)), but now that I think of it, the impression might stem from the static methods in the singleton classes.

Reply:
Yeah that's fair enough, happy for this to be kept static